src/services/utils.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { getUserToken } from '../redux/commonStore.ts';
+
+export const backendFetch = (url: string, init: any, token?: string) => {
+    const initCopy = prepareRequest(init, token);
+    return safeFetch(url, initCopy);
+};
+
+const prepareRequest = (init: any, token?: string) => {
+    if (!(typeof init == 'undefined' || typeof init == 'object')) {
```

Comment:
```suggestion
    if (!(typeof init === 'undefined' || typeof init === 'object')) {
```

Reply:
I kept the same as in gridstudy / explore but yeah I correct it

Reply:
gridstudy code isn't all good 🤣